modules/cluster-logging-collector-pod-location.adoc

```diff
@@ -14,13 +14,12 @@ Run the following command in the `openshift-logging` project:
 ----
 $ oc get pods --all-namespaces -o wide | grep fluentd
```

Comment:
How about use the --selector? 
$oc get pods --selector component=fluentd -o wide -n openshift-logging
NAME            READY   STATUS    RESTARTS   AGE    IP            NODE                  NOMINATED NODE   READINESS GATES
fluentd-8d69v   1/1     Running   0          134m   10.130.2.30   master1.example.com    <none>           <none>
fluentd-bd225   1/1     Running   0          134m   10.131.1.11   master2.example.com    <none>           <none>
fluentd-cvrzs   1/1     Running   0          134m   10.130.0.21   master3.example.com    <none>           <none>
fluentd-gpqg2   1/1     Running   0          134m   10.128.2.27   worker1.example.com    <none>           <none>
fluentd-l9j7j   1/1     Running   0          134m   10.129.2.31   worker2.example.com    <none>           <none>